backend/voting/admin.py

```diff
@@ -110,14 +110,21 @@ def gender(self, obj):
             "other": "🧑🏻‍🎤",
             "not_say": "⛔️",
         }
-        return emoji[obj.submission.speaker.gender]
+        speaker_gender = self._users_by_id[str(obj.speaker_id)]["gender"]
```

Comment:
maybe we can extend the mixin to have a simple `self.get_user_data(user_id)['speaker']`

Reply:
I merged it, you can do `get_user_data`